src/nhs_trivial.jl

```diff
@@ -1,53 +1,30 @@
 @doc raw"""
-    TrivialNeighborhoodSearch{NDIMS}(search_radius, eachpoint)
+    TrivialNeighborhoodSearch{NDIMS}(search_radius, eachpoint, periodic_box = nothing)
 
 Trivial neighborhood search that simply loops over all points.
 
 # Arguments
 - `NDIMS`:          Number of dimensions.
 - `search_radius`:  The uniform search radius.
-- `eachpoint`:   `UnitRange` of all point indices. Usually just `1:n_points`.
+- `eachparticle`:   Iterator for all point indices. Usually just `1:n_points`.
```

Comment:
`eachpoint`?

Reply:
Forgot this one, but it's fixed in #32. Since we're only merging to `dev`, I'd ignore this here.